eventbus/src/test/java/pl/edu/uj/kimage/eventbus/EventA.java

```diff
@@ -0,0 +1,4 @@
+package pl.edu.uj.kimage.eventbus;
+
+public class EventA implements Event {
+}
```

Comment:
I saw it in the next commits, so now it's OK. It does not change the fact that if you needed this class you should add some comments for empty block code